src/plots/cartesian/tick_mark_defaults.js

```diff
@@ -9,14 +9,20 @@ var layoutAttributes = require('./layout_attributes');
  * options: inherits outerTicks from axes.handleAxisDefaults
  */
 module.exports = function handleTickMarkDefaults(containerIn, containerOut, coerce, options) {
-    var tickLen = Lib.coerce2(containerIn, containerOut, layoutAttributes, 'ticklen');
-    var tickWidth = Lib.coerce2(containerIn, containerOut, layoutAttributes, 'tickwidth');
-    var tickColor = Lib.coerce2(containerIn, containerOut, layoutAttributes, 'tickcolor', containerOut.color);
-    var showTicks = coerce('ticks', (options.outerTicks || tickLen || tickWidth || tickColor) ? 'outside' : '');
+    var isMinor = options.isMinor;
+    var cIn = isMinor ? containerIn.minor || {} : containerIn;
+    var cOut = isMinor ? containerOut.minor : containerOut;
+    var lAttr = isMinor ? layoutAttributes.minor : layoutAttributes;
+    var prefix = isMinor ? 'minor.' : '';
+
+    var tickLen = Lib.coerce2(cIn, cOut, lAttr, 'ticklen');
+    var tickWidth = Lib.coerce2(cIn, cOut, lAttr, 'tickwidth');
+    var tickColor = Lib.coerce2(cIn, cOut, lAttr, 'tickcolor', cOut.color);
```

Comment:
Likewise, should these three inherit from major ticks? Len perhaps 60% of the major tick length, width and color the same as the major ticks?

Reply:
See my comment: https://github.com/plotly/plotly.js/pull/6166#discussion_r850411664